canonical-websites/snapcraft.io
# Move template files

Fixes https://github.com/canonical-websites/snapcraft.io/issues/860

In preparation for brand store homepage, organise the template files a little

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004
- Visit http://0.0.0.0:8004/store
- Visit http://0.0.0.0:8004/search?category=featured
- Visit http://0.0.0.0:8004/search?category=&q=skype
- Visit http://0.0.0.0:8004/skype
- Visit http://0.0.0.0:8004/account/snaps
- Visit http://0.0.0.0:8004/account/register-name
- Visit http://0.0.0.0:8004/account/agreement
- Visit http://0.0.0.0:8004/account/username
- Visit http://0.0.0.0:8004/account/details
- Visit http://0.0.0.0:8004/account/snaps/lukewh-test/listing
- Visit http://0.0.0.0:8004/account/snaps/lukewh-test/release
- Visit http://0.0.0.0:8004/account/snaps/lukewh-test/metrics
- Make sure all the pages work as expected
